### PR TITLE
task/tup-664 Alternative ticket resolution - refactor: use core-styles instead of custom css

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview

- Use Core-Styles instead of custom CSS.
- Remove FormikCheckbox commented code.
    <sup>It was commented out because of form validation trouble given the unique requirement in #404 to make "Reply" text area **not** be required if user choses to resolve ticket.</sup>

## Related

- [TUP-664](https://tacc-main.atlassian.net/browse/TUP-664)

## Changes

- **removed** custom css
- **changed** markup to what `c-form` expects
- **removed** commented code

## Testing

See #404.

## UI

https://github.com/TACC/tup-ui/assets/62723358/aa9f63b2-2005-4ca3-99b1-d6f465772aee

### CODE REVIEW

File: libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ interface TicketReplyFormValues {
 
 /* This validates the form for the first textarea in the form.
 This will also show the red required text underneath the textarea.
-If this is implemented, you must also add the `validationSchema={formSchema}` attribute to the Formik component. 
+If this is implemented, you must also add the `validationSchema={formSchema}` attribute to the Formik component.
```
Comment (wesleyboar) [inline libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx:18]: **Superfluous Changes.** Restore space removed upon save of file in my code editor.

```suggestion
If this is implemented, you must also add the `validationSchema={formSchema}` attribute to the Formik component. 
```

File: libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx [hunk 2]
```diff
@@ -104,32 +104,31 @@ export const TicketReplyForm: React.FC<{
...
+                  <label>
+                    <Field type="checkbox" name="status" />
+                    My issue has been resolved
+                  </label>
+                  ) : (
+                  <label>
+                    <Field type="checkbox" name="status" checked disabled />
+                    My issue has been resolved
+                  </label>
+                  )}
```
Comment (wesleyboar) [inline libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx:121]: **Redundant Markup.** Can this be less redundant? Idea first proposed in https://github.com/TACC/tup-ui/pull/404/files#r1447971645.

File: libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx (hunk 1 above)
Comment (wesleyboar) [reply libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx:18]: Same thing, again.

```suggestion
If this is implemented, you must also add the `validationSchema={formSchema}` attribute to the Formik component. 
```

Comment (wesleyboar) [reply libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx:18]: GitHub won't let me do this. But, my editor **should** be removing stray spaces. So, I'm not fighting this extra change in the diff.

**Review by R-Tomas-Gonzalez — APPROVED** [review]
Looks good. Brought up to date with. [#404](https://github.com/TACC/tup-ui/pull/404)

**Pull request merged**